### PR TITLE
support get workload's revision by revision number

### PR DESCRIPTION
**Pull request by richardxz**

### CODE REVIEW

File: pkg/apis/v1alpha/daemonsets/daemonsets.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+	tags := []string{"daemonsets"}
+
+	ws.Route(ws.GET(subPath).To(getDaemonSetRevision).Consumes("*/*").Metadata(restfulspec.KeyOpenAPITags, tags).Doc("Handle daemonset" +
+		" operation").Param(ws.PathParameter("daemonset", "daemonset's name").DataType("string")).Param(ws.PathParameter("namespace",
+		"daemonset's namespace").DataType("string")).Param(ws.PathParameter("revision", "daemonset's revision")).Writes(v1.DaemonSet{}))
+}
+
+func getDaemonSetRevision(req *restful.Request, resp *restful.Response) {
+	deploy := req.PathParameter("daemonset")
```
Comment (wnxn) [inline pkg/apis/v1alpha/daemonsets/daemonsets.go]: Maybe you want to write "daemonset" other than  "deploy"

Comment (richardxz) [reply pkg/apis/v1alpha/daemonsets/daemonsets.go]: yes, i have corrected it

File: pkg/apis/v1alpha/statefulsets/statefulsets.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+	"kubesphere.io/kubesphere/pkg/constants"
+	"kubesphere.io/kubesphere/pkg/models"
+)
+
+func Register(ws *restful.WebService, subPath string) {
+
+	tags := []string{"statefulsets"}
+
+	ws.Route(ws.GET(subPath).To(getDaemonSetRevision).Consumes("*/*").Metadata(restfulspec.KeyOpenAPITags, tags).Doc("Handle deploy" +
```
Comment (wnxn) [inline pkg/apis/v1alpha/statefulsets/statefulsets.go]: Maybe another misspelling about "deploy" occurred here.

Comment (richardxz) [reply pkg/apis/v1alpha/statefulsets/statefulsets.go]: yes , it has been corrected

**Pull request merged**